Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:14.16.0-alpine3.13
```

Comment:
Look at this file for a better example on how to write a docker file for a server:
[Dockerfile](https://github.com/EdgeApp/edge-info-server/blob/Jacob/docker1/Dockerfile)

Reply:
What's specifically is missing from this PR's Dockerfile to the example?

Reply:
Need to add:

```
RUN pm2 install pm2-logrotate

VOLUME [ "./logs" ]
```

Also, the `pm2.json` needs modification to route logs to `./logs`.